### PR TITLE
source/administration/mmctl: update local mode doc to include socketpath

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
`MMCTL_LOCAL_SOCKET_PATH` env variable was missing here 🤦

### CODE REVIEW

File: source/administration/mmctl-cli-tool.rst [hunk 1]
```diff
@@ -101,7 +101,7 @@ To use local mode, the Mattermost server first needs to `have local mode enabled
 Using local mode
 ----------------
 
-You need to append ``--local`` to the command you want to use or set the environment variable as ``MMCTL_LOCAL=true``.
+You need to append ``--local`` to the command you want to use or set the environment variable as ``MMCTL_LOCAL=true``. If you want to use another socket file instead of the default one, you need to set it to ``MMCTL_LOCAL_SOCKET_PATH`` environment variable. Note that this file should be same with the `server configuration <https://docs.mattermost.com/administration/config-settings.html#enable-local-mode-socket-location>`.
```
Comment (cwarnermm) [inline source/administration/mmctl-cli-tool.rst:104]:
```suggestion
You need to append ``--local`` to the command you want to use, or set the environment variable as ``MMCTL_LOCAL=true``. To use a socket file other than the default, you need to set the environment variable to ``MMCTL_LOCAL_SOCKET_PATH``. This file should match the `server configuration setting <https://docs.mattermost.com/administration/config-settings.html#enable-local-mode-socket-location>`.
```

Comment (cwarnermm) [reply source/administration/mmctl-cli-tool.rst:104]: @isacikgoz When using a socket file other than the default, we're currently saying that the file _should_ match the `local mode socket location` server configuration setting. Are we making a recommendation or is this a requirement? If it's the latter, I'd like to strengthen the language from "should" to "must".

**Review by cwarnermm — COMMENTED** [review]
Great addition! Thank you! Suggested update and an open question provided inline.

File: source/administration/mmctl-cli-tool.rst (hunk 1 above)
Comment (isacikgoz) [reply source/administration/mmctl-cli-tool.rst:104]: Yep this is a requirement, we can strengthen the wordage there 👍

Comment (cwarnermm) [reply source/administration/mmctl-cli-tool.rst:104]: Thanks! Suggestion updated!

Comment (isacikgoz) [reply source/administration/mmctl-cli-tool.rst:104]: Awesome! thanks @cwarnermm 🎉

**Review by cwarnermm — APPROVED** [review]
Thanks!

File: source/administration/mmctl-cli-tool.rst [hunk 2]
```diff
@@ -103,7 +103,7 @@ To use local mode, the Mattermost server first needs to `have local mode enabled
 Using local mode
 ----------------
 
-You need to append ``--local`` to the command you want to use or set the environment variable as ``MMCTL_LOCAL=true``.
+You need to append ``--local`` to the command you want to use, or set the environment variable as ``MMCTL_LOCAL=true``. To use a socket file other than the default, you need to set the environment variable to ``MMCTL_LOCAL_SOCKET_PATH``. This file must match the `server configuration setting <https://docs.mattermost.com/administration/config-settings.html#enable-local-mode-socket-location>`.
```
Comment (cwarnermm) [inline source/administration/mmctl-cli-tool.rst]:
```suggestion
You need to append ``--local`` to the command you want to use, or set the environment variable as ``MMCTL_LOCAL=true``. To use a socket file other than the default, you need to set the environment variable to ``MMCTL_LOCAL_SOCKET_PATH``. This file must match the `server configuration setting <https://docs.mattermost.com/administration/config-settings.html#enable-local-mode-socket-location>`__.
```

Comment (justinegeffen) [inline source/administration/mmctl-cli-tool.rst:106]:
```suggestion
You need to append ``--local`` to the command you want to use, or set the environment variable as ``MMCTL_LOCAL=true``. To use a socket file other than the default, you need to set the environment variable to ``MMCTL_LOCAL_SOCKET_PATH``. This file must match the `server configuration setting <https://docs.mattermost.com/administration/config-settings.html#enable-local-mode-socket-location>`_.
```

Comment (cwarnermm) [reply source/administration/mmctl-cli-tool.rst]: Updated required formatting for docs link.

**Review by justinegeffen — APPROVED** [review]

**Review by cwarnermm — CHANGES_REQUESTED** [review]
Looks great! Formatting fix provided for the docs link.

File: source/administration/mmctl-cli-tool.rst (hunk 2 above)
Comment (justinegeffen) [reply source/administration/mmctl-cli-tool.rst]: Sorted. :)

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1:

**Review by cwarnermm — APPROVED** [review]
Thank you!

**Pull request merged**